test/integration/dialects/mariadb/query-interface.test.js

```diff
@@ -1,24 +0,0 @@
-'use strict';
-
-const chai = require('chai');
-
-const expect = chai.expect;
-const Support = require('../../support');
-
-const dialect = Support.getTestDialect();
-
-if (dialect.startsWith('mariadb')) {
-  describe('QueryInterface', () => {
-
-    describe('databases', () => {
-      it('should create and drop database', async function () {
```

Comment:
replaced with `create-drop-list-database.test.ts`, which runs on all dialects instead of just mariadb